### ISSUE TITLE
Kava 0.16.0 crashes if swagger for api is enabled

### ISSUE THREAD

**Issue by Krewedk0** [Kava-Labs/kava#1146]
If `swagger` for api is enabled in `app.toml`, node crashes on start with following error

`panic: TODO: register swagger in app`

**Comment by jun0tpyrc** [comment]
same 
```
panic: TODO: register swagger in app

goroutine 1 [running]:
github.com/kava-labs/kava/app.(*App).RegisterAPIRoutes(0x22ff648, 0xc000c68380, {0x1, 0x1, 0x0, {0xc000161660, 0x12}, 0x3e8, 0x1e, 0x0, ...})
        /root/kava/app/app.go:790 +0x392
github.com/cosmos/cosmos-sdk/server.startInProcess(_, {{0x0, 0x0, 0x0}, {0x2339950, 0xc0099caa00}, {0x0, 0x0}, {0x22ff648, 0xc000fcec30}, ...}, ...)
        /go/pkg/mod/github.com/kava-labs/cosmos-sdk@v0.44.5-kava.1/server/start.go:305 +0xbb3
github.com/cosmos/cosmos-sdk/server.StartCmd.func2(0xc001022f00, {0xc00016a900, 0x0, 0x4})
        /go/pkg/mod/github.com/kava-labs/cosmos-sdk@v0.44.5-kava.1/server/start.go:129 +0x193
github.com/spf13/cobra.(*Command).execute(0xc001022f00, {0xc00016a8c0, 0x4, 0x4})
        /go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:856 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0xc000c7db80)
        /go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:974 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
        /go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:902
github.com/spf13/cobra.(*Command).ExecuteContext(...)
        /go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:895
github.com/cosmos/cosmos-sdk/server/cmd.Execute(0x0, {0xc000fa9230, 0xb})
        /go/pkg/mod/github.com/kava-labs/cosmos-sdk@v0.44.5-kava.1/server/cmd/execute.go:36 +0x1eb
```

**Comment by karzak** [comment]
Ah, that's a bug we forgot to clean up in v0.16. In ~/.kava/config/app.toml set `swagger=false` and restart node

**Comment by Galadrin** [comment]
The issue is always valid for kava 0.17.0-alpha.0

**Comment by joslee7410** [comment]
> Ah, that's a bug we forgot to clean up in v0.16. In ~/.kava/config/app.toml set `swagger=false` and restart node. We'll patch and release v0.16.1

I think it's not yet fix right? Was running with v0.16.1 but swagger still unable to open.
